### DISCUSSION TITLE
Office_S_Tips/csv/2019/09/21/pliki_tekstowe_CSV_otwierane_w_Excelu

### DISCUSSION THREAD

**Discussion by giscus** [andrzejQ/Office_S_Tips#22] [Komentarze]
# Office_S_Tips/csv/2019/09/21/pliki_tekstowe_CSV_otwierane_w_Excelu

Excel-CSV - kilka jawnych, ale też kilka mniej znanych możliwości.

https://andrzejq.github.io/Office_S_Tips/csv/2019/09/21/pliki_tekstowe_CSV_otwierane_w_Excelu.html

**Comment by andrzejQ** [comment]
Nowa wersja komentarzy z użyciem `giscus -> GitHub Discussions`.
